Google.PowerShell/BigQuery/BqTable.cs

```diff
@@ -330,4 +330,163 @@ public TablesResource.InsertRequest makeInsertReq()
             return Service.Tables.Insert(newTable, Project, DatasetId);
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Deletes the specified table.
+    /// </para>
+    /// <para type="description">
+    /// Deletes the specified table from the dataset. The table to be deleted can be passed 
+    /// in via the pipeline or with the -InputObject parameter. If the table contains data, 
+    /// this operation will prompt the user for confirmation before any deletions are performed. 
+    /// To delete a non-empty table automatically, use the -Force parameter. If no Project is 
+    /// specified, the default project will be used. This cmdlet returns a Table object. 
+    /// This cmdlet supports the ShouldProcess function, so it has the corresponding -WhatIf 
+    /// and -Confirm flags.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> $table = Get-BqTable -Dataset "my_dataset" -Table "my_table"
+    ///   PS C:\> $table | Remove-BqTable</code>
+    ///   <para>This will remove "my_table" if it is empty, and will prompt for user confirmation 
+    ///   if it is not. All data in "my_table" will be deleted if the user accepts.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Remove-BqTable -DatasetId "my_dataset" -Table "my_table" -Force</code>
+    ///   <para>This will remove "my_table" and all of its data.</para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/tables)">
+    /// [BigQuery Tables]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "BqTable", SupportsShouldProcess = true)]
+    
+    public class RemoveGcbqTable : BqCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string ByValue = "ByValue";
+            public const string ByDatasetObject = "ByDatasetObject";
+            public const string ByObject = "ByObject";
+        }
+
+        /// <summary>
+        /// <para type="description">
+        /// The project to look for tables in. If not set via PowerShell parameter processing, it will
+        /// default to the Cloud SDK's DefaultProject property.
```

Comment:
DefaultProject is not the name of the property (technically it is core/project). You could just say "the Cloud SDK's default project." We can assume our users will know what that means.